src/Service/Resolvers/MsSqlQueryBuilder.cs

```diff
@@ -61,10 +61,12 @@ public string Build(SqlQueryStructure structure)
         /// <inheritdoc />
         public string Build(SqlInsertStructure structure)
         {
-
+            string predicates = JoinPredicateStrings(structure.DbPolicyPredicates);
```

Comment:
What is the predicates string when DbPolicyPredicates doesnt contain a predicate for create operation? 

Do we get a 1=1 predicate? 

Do we need to create the SELECT FROM WHERE construct for that situation - wouldnt a simple insert into suffice? - will there be performance hit if we create a temp table even when we dont need it?

Reply:
Yes, we get 1=1. Yeah, there would be a performance hit. We can have a check in place for this.